example-github/src/main/java/feign/example/github/GitHubExample.java

```diff
@@ -53,4 +64,51 @@ public static void main(String... args) throws InterruptedException {
     String login;
     int contributions;
   }
+
+  static class ClientError {
```

Comment:
Gson should probably work as it uses sun.misc.unsafe

Give it a shot! Main goal is to show the fast path, and I think declaring
one type helps minimize distraction.

If doesn't work, I apologize in advance ;)